src/utils/utils.ts

```diff
@@ -27,3 +16,48 @@ export function getNextSortValue(value?: "asc" | "desc" | null) {
       return "asc";
   }
 }
+
+export function filterData<T>(
+  data: T[],
+  column: Filter,
```

Comment:
Why `column` is `Filter`? It's either a column (I'd expect a column to be an obj describing the column: id, name, label), or a filter (key & value)

Reply:
yap you are correct!